Parser.py

```diff
@@ -257,20 +244,14 @@ def extract_constrain_coefficients(modified_lines, range1):
 def extract_right_parts(modified_lines):
     list = []
     for m in range(3, len(modified_lines) - 1):
-        if re.findall(">=", modified_lines[m]):
-            index = modified_lines[m].find("=")
-        elif re.findall("=", modified_lines[m]):
-            index = modified_lines[m].find("=")
-        elif re.findall("<=", modified_lines[m]):
-            index = modified_lines[m].find("=")
+        index = modified_lines[m].find("=")
         right_part = modified_lines[m][index + 1:]
-        list.append(right_part)
+        list.append(int(right_part))

```

Comment:
ValueError: invalid literal for int() with base 10: '-8,'

works well if int() is removed and reverted to original form

Reply:
It works on me. Where did it find the comma after the 8? Commas where removed from the modified file (see line 41 from the modified).

> works well if int() is removed and reverted to original form

If we remove `int()`, the number will be stored as a string; something we definitely not want.

Reply:
I did made the changes but somehow it gives me a warning

But that should be right

Reply:
I did made the changes but somehow it gives me a warning

But that should be right